code/game/objects/structures/false_walls.dm

```diff
@@ -266,6 +266,20 @@
 	walltype = /turf/closed/wall/mineral/bananium
 	canSmoothWith = list(/obj/structure/falsewall/bananium, /turf/closed/wall/mineral/bananium)
 
+/obj/structure/falsewall/tranquillite
+	name = "tranquillite wall"
+	desc = "A wall with tranquillite plating. Nearly invisible."
+	icon = null
+	icon_state = null
+	mineral = /obj/item/stack/sheet/mineral/tranquillite
+	walltype = /turf/closed/wall/mineral/tranquillite
+	canSmoothWith = list(/obj/structure/falsewall/tranquillite, /turf/closed/wall/mineral/tranquillite)
+	opacity = 0 //actually invisible
+
+/obj/structure/falsewall/tranquillite/toggle_open()
+	..()
+	opacity = 0
+	update_icon()
```

Comment:
Can I ask how this works? I have an idea but I'm kinda just prodding you to see if you tested this.

Reply:
It works as in [WIP] and "Probably should make Tranquillite walls not completely invisible." aka "Probably should make Tranquillite walls not completely invisible and then test them after they're considered complete instead of before."

Reply:
Right now you'd have to alt click (maybe) to open and close them but I think a more viable option would be to use the normal wall icons and just set alpha and opacity so it's otherwise hard to see.

Reply:
I really need to fuck with the walls as a whole because I forgot that walls won't show the baseturf so I have to do something like indestructible glass and add a fake plating on Initialize()